### ISSUE TITLE
Profile results do not match Bill's

### ISSUE THREAD

**Issue by mollybsmith-noaa** [NOAA-GSL/MATS#46]
---

Author Name: **kevin.bruhwiler** (kevin.bruhwiler)
Original Redmine Issue: 21246, https://vlab.ncep.noaa.gov/redmine/issues/21246
Original Date: 2016-08-12

---

The data returned by the query is sometimes a little off, sometimes very different. Graphs look similar, sometimes identical, but can be different in small ways. The query looks the same, and I’m unsure if the different numbers for the data is intentional.

1. Plot a TimeSeries or Profile plot with both the MATS and the old RUC site
2. Make sure both are plotting the same graph and that the RUC ‘show text’ box is checked
3. Compare the data of the two plots. Changing any of the parameters will affect the data, however the FIM model over the RUC region with
    temperature as the variable will return obvious differences.

---

- [MATS-test-20170306-compare.JPG](https://vlab.ncep.noaa.gov/redmine/attachments/download/7527/MATS-test-20170306-compare.JPG) (bonny.strong)

**Comment by mollybsmith-noaa** [comment]
---

Original Redmine Comment
Author Name: **randy.pierce** (@randytpierce)
Original Date: 2016-08-17T20:29:28Z

---

testing is manual for now.

**Comment by mollybsmith-noaa** [comment]
---

Original Redmine Comment
Author Name: **bonny.strong** (@bonnystrong)
Original Date: 2017-03-06T23:10:53Z

---

Test case comparing MATS against Bill's shows definite different results.

Using Kevin's suggested values, used model=FIM and region=RUC and compared text.  See attached screenshot.
Look especially at 1000 hPa.

---

- [MATS-test-20170306-compare.JPG](https://vlab.ncep.noaa.gov/redmine/attachments/download/7527/MATS-test-20170306-compare.JPG) (bonny.strong)

**Comment by mollybsmith-noaa** [comment]
---

Original Redmine Comment
Author Name: **bonny.strong** (@bonnystrong)
Original Date: 2017-03-10T19:10:58Z

---

Ach!  Looked closer at the comparison snapshot that I included.  And I see that I was comparing the 0-hr forecast from Bill's app to the 12-hr forecast in MATS.

Checked now with 12-hr to 12-hr and they match exactly.

Closing this issue now, but will continue testing.  If any other differences come up, will open a new issue.
